src/views/UIOptionsPopup/style.ts

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components'
+
+export const UIOptionsDialog = styled.div`
+  top: 40px;
```

Comment:
I'd make this 43px to give it more of a floating sense.

Reply:
Done